api.py

```diff
@@ -11,7 +11,7 @@
 
 from datetime import datetime, date, timedelta
 from dateutil.parser import parser as date_parser
-from flask import Flask, abort, request, flash, redirect, render_template, url_for, jsonify, got_request_exception
+from flask import Flask, abort, request, flash, redirect, render_template, url_for, jsonify, got_request_exception, g
```

Comment:
http://flask.pocoo.org/docs/0.10/api/#flask.g

I use it to store the account after validating the API KEY. In the method we then can access the site id etc. from the account DB entry.